static/styles/settings.css

```diff
@@ -1236,6 +1236,11 @@ input[type="checkbox"] {
     }
 }
 
+#change_user_group_name,
+#change_user_group_description {
+    width: 100%;
+}
+
```

Comment:
This should be in the same commit adding the modal behavior.

Reply:
Made this change in the updated commits.